adam_core/dynamics/kepler.py

```diff
@@ -6,6 +6,7 @@
 from .barker import solve_barker
 
 config.update("jax_enable_x64", True)
+config.update("jax_platform_name", "cpu")
```

Comment:
Can jax do automatic platform detection? Guessing we want to set this intentionally somewhere at runtime.

Reply:
Yes, it does by default. This is just a simple workaround for it to stop complaining about not finding a TPU/GPU to run on. We can remove these statements once we start running on more than just CPUs.